pkg/management/barman/backuplist.go

```diff
@@ -133,5 +139,5 @@ func GetBackupList(
 		return nil, err
 	}
 
-	return backupList, nil
+	return &backupList, nil
```

Comment:
no need for the pointer to be returned here now, the function should return a catalog.Catalog directly